manifests/init.pp

```diff
@@ -137,6 +137,8 @@
 #   Additional config variables in rabbitmq.config
 # @param config_cluster
 #   Enable or disable clustering support.
+# @param config_cowboy_opts
+#   Additional config variables for cowboy_opts in rabbitmq.config.
```

Comment:
probably note that this is a hash of values, similar to `$config_kernel_variables` etc.